### DISCUSSION TITLE
giada-v0.23.0-x86_64-macos version damaged on macOS 12.6 (Monterey)

### DISCUSSION THREAD

**Discussion by bwagner** [monocasual/giada#615] [Q&A]
`giada-v0.23.0-x86_64-macos` version damaged on macOS 12.6 (Monterey).

When running it, I get a dialog: `“giada.app” is damaged and can’t be opened. You should move it to the Bin.`

running it from the command line doesn't provide further insight:
`$ ./giada.app/Contents/MacOS/giada` opens the same dialog, no logging in the shell.

**Comment by gvnnz** [comment]
Hi @bwagner , yes this is "expected" because we are shipping unsigned binaries. Please take a look at the "Problems with the macOS version?" section on the Giada website's [Dowload area](https://www.giadamusic.com/download), you will find a couple of links that will guide you in the right direction.
